### PR TITLE
some browsers do not offer full reload compatibility with the (Comman…

### PR DESCRIPTION
**Pull request by rafaelmilanibarbosa**
…dOrControl + Shift + R) command. The command (CommandOrControl + R) should resolve.

@RocketChat/DesktopApp 

Closes #ISSUE_NUMBER

### CODE REVIEW

File: src/scripts/menus/view.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ const viewTemplate = [
         type: 'separator'
     },
     {
-        label: i18n.__('Application_Reload'),
+        label: i18n.__('Application_Reload'),
```
Comment (gdelavald) [inline src/scripts/menus/view.js]: @rafaelmilanibarbosa Is this the only change or am I missing something?

**Comment by rafaelmilanibarbosa** [comment]
it's just the same

**Comment by gdelavald** [comment]
@rafaelmilanibarbosa you just added a space, nothing else, is it missing code to push or that's it?

**Comment by rafaelmilanibarbosa** [comment]
do not. excuse. I did another commit then but I think the PR did not get it.

**Comment by rafaelmilanibarbosa** [comment]
https://github.com/RocketChat/Rocket.Chat.Electron/pull/566/commits/284a928601d9a70817cc7f01e0c606a3a54f63f7

**Comment by rafaelmilanibarbosa** [comment]
I did only on one element, however the correct one would be to do at all request

**Pull request closed without merging**